auth-tokens-filter/src/main/java/com/palantir/tokens/auth/http/BearerTokenLoggingFilter.java

```diff
@@ -41,17 +43,34 @@
     public static final String SESSION_ID_KEY = "sessionId";
     public static final String TOKEN_ID_KEY = "tokenId";
 
+    private final String cookieAuthKey;
```

Comment:
Conjure allows endpoints to specify different cookie names, how do we plan to register a separate filter instance for each endpoint?

Reply:
I believe we fall on convention here - just like I could define authorization to be under different auth headers. Unless you want to get in the game of generating this logging handler for every endpoint separately but right now our server infra doesn't allow that in all cases and you have already handled the other case

Reply:
>  just like I could define authorization to be under different auth headers

Conjure only allows the `Authorization` to be used for header auth.

> Unless you want to get in the game of generating this logging handler for every endpoint separately

I already am, our undertow handlers have already solved this problem  ;-) 

I think we can solve this for jersey using a DynamicFeature to check the first arg for an auth cookie, without requiring per-app customization or additional code generation.

Reply:
Sure, while I agree with you I don't think logging is covered under spec in any form. Once it added we can adjust it. You'll notice that it's only our server implementation uses this filter and not the conjure-java-runtime part. 

Don't get me wrong - I think what you're proposing is probably the right thing to do just don't think it's valuable right now